src/Pages/Profile/MemberView/Profile.js

```diff
@@ -110,18 +110,18 @@ export default class Profile extends Component {
             user={{ ...this.state.user, token: this.props.user.token }}
           />,
           icon: <FontAwesomeIcon icon={faLock} />
-        }
+        },
       ]
       : [
+        { title: 'Become a member to access this page!', value: '' },
+        { title: '', value: '' },
+        { title: '', value: '' },
+        { title: '', value: '' },
+        { title: '', value: '' },
+        { title: '', value: '' },
+        { title: '', value: '' },
         { title: '', value: '' },
-        { title: 'Door Code', value: '' },
-        { title: 'Joined Date', value: '' },
-        { title: 'Email', value: '' },
-        { title: 'Membership Expiration', value: '' },
-        { title: '2D Prints', value: '' },
-        { title: 'Request 3D Printing', value: '' },
-        { title: 'Sign-in with Discord', value: '' },
-        { title: 'Change Password', value: '' }
+        { title: '', value: '' }
```

Comment:
instead of empty title and value can this array just be 1 element? or does it not work

Reply:
it fails the test if it is not 1 element

Reply:
maybe change the test instead? up to u i already approved

Reply:
sick bet